modules/TwitterStreamingImporter/src/main/resources/totetmatt/gephi/twitter/Bundle.properties

```diff
@@ -0,0 +1,46 @@
+# To change this license header, choose License Headers in Project Properties.
+# To change this template file, choose Tools | Templates
+# and open the template in the editor.
+
+CTL_SimpleAction=Twitter Stream
+CTL_SimpleTopComponent=Twitter Stream
+HINT_SimpleTopComponent=WHEEEEEEE
+OpenIDE-Module-Display-Category=Plugin
```

Comment:
You already put this information in the `manifest.mf` file and you can remove this, as well as the lines below.